.github/workflows/check-api-compatibility.yml

```
@@ -22,21 +22,19 @@ jobs:
           java-version: '11'
           cache: 'maven'
 
-      - name: Build old version
-        run: |
-          cd gson-old-japicmp
-          # Set dummy version
-          mvn --batch-mode --no-transfer-progress org.codehaus.mojo:versions-maven-plugin:2.11.0:set -DnewVersion=JAPICMP-OLD
-          # Install artifacts with dummy version in local repository; used later by Maven plugin for comparison
-          mvn --batch-mode --no-transfer-progress install -DskipTests
+      - name: Set dummy version
```

Maybe the "used later by Maven plugin for comparison" should be preserved in some way (e.g. as comment) to make it clearer why this is done.

What about this?
```suggestion
      # First sets a dummy project version, then installs the artifacts with that version in the local
      # Maven repository, and in a later step uses it as 'old version' for the API compatibility check.
      - name: Set dummy version
```